src/pages/Notification/NotificationPage.jsx

```diff
@@ -70,32 +185,71 @@ const NotificationPage = () => {
           onTagChange={setTag}
         />
         <SBtnWrapper>
-          {filteredNotifications.map((notification, index) => (
-            <NotificationItem
-              key={index}
-              image={notification.image}
-              name={notification.name}
-              message={notification.message}
-              time={notification.time}
-              onClick={() =>
-                notification.type === '친구'
-                  ? friendNotiClick(notification.image, notification.name)
-                  : fundingNotiClick()
-              }
-            />
-          ))}
+          {filteredNotifications.map((notification, index) =>
+            //tag가 친구일때
+            tag === '친구' ? (
+              <SALayout
+                key={index}
+                onClick={() =>
+                  friendNotiClick(
+                    notification.recieveUserImgUrl,
+                    notification.name,
+                    notification.friendTableId,
+                  )
+                }
+              >
+                {notification.recieveUserImgUrl === 'default' ? (
+                  <AStyledProfileIcon />
+                ) : (
+                  <SAImg src={notification.recieveUserImgUrl} />
+                )}
+                <STextContainer>
+                  <STextboxContainer>
+                    <SNameText>{notification.name} </SNameText>
+                    <SMessageText>님이 친구 요청을 보냈습니다</SMessageText>
+                  </STextboxContainer>
+                  <STimeText>{notification.time}</STimeText>
+                </STextContainer>
+              </SALayout>
+            ) : (
+              //tag가 펀딩일때
+              <SALayout onClick={() => fundingNotiClick} key={index}>
+                <SAImg src={notification.fundingImageUrl} />
+                <STextContainer>
+                  <STextboxContainer>
+                    <SNameText>{notification.name} </SNameText>
+                    <SMessageText>
+                      펀딩 {notification.percent}% 달성
+                    </SMessageText>
+                  </STextboxContainer>
+                  <STimeText>{notification.time}</STimeText>
+                </STextContainer>
+              </SALayout>
+            ),
+          )}

```

Comment:
이 부분은 `NotificationItem.jsx`으로 분리하는 게 맞아요! 데이터 넘겨주고 NotificationItem 컴포넌트에서 item의 tag값에 따라 친구 요청 모달창, 알림 텍스트 문구 조건부 렌더링을 처리합니다.
```suggestion
          {notificationList.map((item, index) => (
            <li key={index}>
              <NotificationItem data={item} />
            </li>
          ))}
```